docs/data-sources/agent_token.md

```diff
@@ -8,8 +8,23 @@ description: |-
 
 # traceable_agent_token (Data Source)
 
+## Example Usage
 
+```terraform
+data "traceable_agent_token" "example" {
+  name = "tf-provider-token-testing"
+}
 
+output "agent_token" {
+  value = data.traceable_agent_token.example.token
+  sensitive = true
```

Comment:
is it still required as we removed the value from schema?

Reply:
its optional, this doesnt correspond to the field in the schema, its just a part of the output block indicating the show of sensitive info or to hide it